### PR TITLE
EfficientNet B0 to B5 image classification templates for Caffe

### PR DESCRIPTION
**Pull request by beniz**
Pre-trained models are available from https://www.deepdetect.com/models/efficientnet
Input images require a pre-scaling value around 0.0083. 
The pretrained models may not be too useful outside of transfer learning tasks, due to incompatibilities when converting from TF, and they require an adapted scaling factor. Additionnally, it's proved useful in certain cases to freeze the batchnorm layers for transfer learning.

### CODE REVIEW

**Comment by BirdeeChen** [comment]
Thanks for the excellent work, can you please upload the prototxt as well?

**Comment by beniz** [comment]
The prototxt are part of this PR, you can download them easily.
